### PR TITLE
Combine poll commands

### LINKED ISSUE #47: Merge poll commands into one ;poll
Instead of having `;abc` and `;yesno` commands, we rather should do one command `;poll` that takes 1 parameter (`;poll "question"`) if that's a yes/no poll, that will make 👍 and 👎 reactions, or 1+n parameters (`;poll "question" "param1" "param2", ...`) that will make A, B, C, ... reactions.

Also, we should force syntax to require quotes for parameters, to make the API clear.

Benefits:
- simpler syntax
- clearer name
- ...

### PR DESCRIPTION
**Pull request by albertmoravec**
This should hopefully work the same way as the previous two commands did, but I didn't test it.

Closes #47

### CODE REVIEW

File: src/HonzaBotner.Discord.Services/Commands/Polls/PollCommand.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        }
+
+        protected List<string> ParseParameters(string message)
+        {
+            if (message.Count(c => c == '"') % 2 != 0)
+            {
+                throw new ArgumentException("Odd number of quotes");
+            }
+
+            const string quotedGroupPattern = @"""([^\""]*?)""";
```
Comment (ostorc) [inline src/HonzaBotner.Discord.Services/Commands/Polls/PollCommand.cs:65]: nit: Maybe add comment explaining this regex...

**Review by ostorc — APPROVED** [review]

File: src/HonzaBotner.Discord.Services/Commands/Polls/AbcPoll.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.Entities;
+
+namespace HonzaBotner.Discord.Services.Commands.Polls
+{
+    public class AbcPoll : IPoll
+    {
+        private static readonly List<string> _optionsEmoji = new List<string>
```
[the remark this answers is not in the document]
Comment (albertmoravec) [reply src/HonzaBotner.Discord.Services/Commands/Polls/AbcPoll.cs:12]: Definitely doable, I just thought it was meant as a hard answers limit in the previous implementation.

[the remark this answers is not in the document]
Comment (ostorc) [reply src/HonzaBotner.Discord.Services/Commands/Polls/AbcPoll.cs:12]: Not sure why it was like that. I didn't really looked at the blob of deleted files.. I don't really care how many letters there will be, 10 is in my opinion more than enough.

[the remark this answers is not in the document]
Comment (albertmoravec) [reply src/HonzaBotner.Discord.Services/Commands/Polls/AbcPoll.cs:12]: Set it to 16 for now and left the rest commented out... we can just uncomment it if desirable.

File: src/HonzaBotner.Discord.Services/Commands/Polls/PollCommand.cs (hunk 1 above)
Comment (albertmoravec) [reply src/HonzaBotner.Discord.Services/Commands/Polls/PollCommand.cs:65]: Sure, it's kinda cryptic.

File: src/HonzaBotner.Discord.Services/Commands/Polls/AbcPoll.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (tenhobi) [reply src/HonzaBotner.Discord.Services/Commands/Polls/AbcPoll.cs:12]: It was 20 cause there could be only 20 reactions in total.

**Comment by tenhobi** [comment]
```
2020-10-31T13:40:46.296450+00:00 app[web.1]: [40m[32minfo[39m[22m[49m: HonzaBotner.Discord.DiscordBot[0]
2020-10-31T13:40:46.296460+00:00 app[web.1]:       poll
2020-10-31T13:40:46.296468+00:00 app[web.1]: [40m[32minfo[39m[22m[49m: HonzaBotner.Discord.DiscordBot[0]
2020-10-31T13:40:46.296469+00:00 app[web.1]:       HonzaBotner.Discord.Services.Commands.Polls.PollCommand
2020-10-31T13:40:46.296738+00:00 app[web.1]: [41m[30mfail[39m[22m[49m: DSharpPlus.DiscordClient[0]
2020-10-31T13:40:46.296739+00:00 app[web.1]:       [DSharpPlus]: An System.AggregateException occured in MESSAGE_CREATED.
```

**Pull request merged**